src/Nancy.Swagger/SwaggerExtensions.cs

```diff
@@ -324,15 +324,25 @@ public static Schema GetSchema<T>(ISwaggerModelCatalog modelCatalog)
 
         public static Schema GetSchema(ISwaggerModelCatalog modelCatalog, Type t)
         {
+            if (SwaggerTypeMapping.IsMappedType(t))
+            {
+                t = SwaggerTypeMapping.GetMappedType(t);
+			}

```

Comment:
Ah. I was using a different computer/Visual Studios and I realized afterwards that I was inputting tabs and not spaces. I thought I fixed them, but I guess I missed one.